.github/workflows/wheel_win_x86_64.yml

```diff
@@ -148,7 +148,7 @@ jobs:
             pip install pybind11 cmake~=3.24.0 build
 
           # Testing of built wheels
-          CIBW_TEST_REQUIRES: numpy~=1.21 scipy pytest pytest-cov pytest-mock flaky
+          CIBW_TEST_REQUIRES: numpy~=1.23 scipy pytest pytest-cov pytest-mock flaky
```

Comment:
Is there any reason for using this instead of the <1.24 used in the other workflow files?